03.2_school_prepare_OD_file.R

```diff
@@ -8,14 +8,27 @@ geography <- "lsoa"
 set.seed(12345)
 
 #########################
-# LOAD SCHOOL FLOW DATASET
+# LOAD SCHOOL FLOW DATASETS
 #########################
-sf11 <- read_tsv(file.path(path_inputs, "01_raw/02_travel_data/school/NPD_originals_PRIVATE/Spring_Census_2011.txt"))
-sf11 <- sf11[,c(1, 3:4, 6:10)]
-names(sf11) <- sub(paste0("_SPR11"), "", names(sf11))
-names(sf11) <- tolower(names(sf11))
-sf11 <- dplyr::rename(sf11, lsoa01cd = `llsoa`, bicycle = `cycle`, foot = `walk`)
+# Open and rename the 2010 and 2011 datasets
+for (i in (10:11) ) {
+  sf <- read_tsv(file.path(path_inputs, "01_raw/02_travel_data/school/lsoa/NPD_originals_PRIVATE", paste0("Spring_Census_20",i,".txt")))
+  sf <- sf[,c(1, 3:4, 6:10)]
+  names(sf) <- sub(paste0("_SPR",i), "", names(sf))
+  names(sf) <- tolower(names(sf))
+  sf <- dplyr::rename(sf, lsoa01cd = `llsoa`, bicycle = `cycle`, foot = `walk`)
```

Comment:
Back-ticks are not required around variable names in dplyr verbs unless the names contain non-standard characters, e.g. `cycle path` would require back-ticks because it has a space.

Reply:
okay, I didn't know that – I think I just been copying this code from something Malcolm originally wrote, perhaps some of those variables were non-standard or he was just including them to be safe. I have removed them